aws/AWS-CloudFormation-Template-ZipUpload.yaml

```diff
@@ -0,0 +1,111 @@
+AWSTemplateFormatVersion: 2010-09-09
+Metadata:
+  'AWS::CloudFormation::Designer':
+    45e858b2-e2f1-43f4-8836-5ad4c06a5500:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -130
+        'y': 120
+      z: 0
+      embeds: []
+    f41d472a-361a-4dc7-b9dc-612ad6924bd1:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -130
+        'y': 40
+      z: 0
+      embeds: []
+    c7e02f6b-ec1f-4f55-bb0a-e882661c0c34:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -230
+        'y': 40
+      z: 0
+      embeds: []
+    47377fcc-9f07-408a-885a-310f0712c2df:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -10
+        'y': 80
+      z: 0
+      embeds: []
+Description: Resources for ISIC Archive ZIP upload
+Resources:
+  ZipUploadBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties: {}
```

Comment:
How does the name get chosen? Automatically by CloudFormation? We really are only going to provision a singleton bucket in production.

Reply:
Let's try using the name `isic-data-upload-${AWS::StackName}`. We'll still need to output the final value.

Reply:
I append the stack name to a couple other resource names now as well. This gives us descriptive names, as opposed to the generated names, while avoiding name collisions when creating multiple stacks from the template.